### PR TITLE
make ext test need for existing cluster painfully obvious

### PR DESCRIPTION
**Pull request by gabemontero**
@smarterclayton per recent email thread

@openshift/sig-developer-experience fyi

### CODE REVIEW

File: test/extended/setup.sh [hunk 1]
```diff
@@ -40,6 +40,11 @@ function os::test::extended::focus () {
 #		and then tests are executed.	Tests that depend on fine grained setup should
 #		be done in other contexts.
 function os::test::extended::setup () {
+        echo ""
+        os::log::info "REMINDER, EXTENDED TESTS NO LONGER START A CLUSTER."
```
[the remark this answers is not in the document]
Comment (gabemontero) [reply test/extended/setup.sh]: switch  to warning pushed

**Comment by gabemontero** [comment]
bump @smarterclayton @bparees ... the new log now leverages os::log::warning

File: test/extended/setup.sh [hunk 2]
```diff
@@ -40,6 +40,11 @@ function os::test::extended::focus () {
 #		and then tests are executed.	Tests that depend on fine grained setup should
 #		be done in other contexts.
 function os::test::extended::setup () {
+        echo ""
+        os::log::warning "REMINDER, EXTENDED TESTS NO LONGER START A CLUSTER."
+        os::log::warning "THE CLUSTER REFERENCED BY THE 'KUBECONFIG' ENV VAR IS USED."
+        echo ""
```
Comment (bparees) [inline test/extended/setup.sh]: align the indentation please

**Comment by gabemontero** [comment]
indentation adjusted

**Pull request merged**